rust/template/distributed_datalog/src/schema/relations.rs

```diff
@@ -0,0 +1,21 @@
+use std::collections::BTreeSet;
```

Comment:
Do we really need a file per data structure? I expect this schema to change a lot initially (and eventually get transformed into a DDlog schema); you'll waste a lot of effort adding/removing/renaming these files.  This also makes it hard to see big picture compared to having all schema data structure in one place, declared in a bottom-up (or top-down) order.

Reply:
It seemed a bit too much for a single file. But I am not objecting. Will change it.